.github/workflows/build-msvc.yml

```diff
@@ -60,6 +60,8 @@ jobs:
 
       - name: Install Cygwin
         uses: cygwin/cygwin-install-action@v3
+        env:
+          CYGWIN: winsymlinks:nativestrict
```

Comment:
Does this definitely work (have you checked the directory listing after installation?) I thought it remains necessary to pass `--symlink-type native` (cf. https://github.com/ocaml/opam/pull/5830). Is it significantly affecting the cache?

Reply:
Hmm, I'm not sure - I have memories in the (not too distant) past of symlinks causing strange issues elsewhere (while doing [dra27/cygsympathy](https://github.com/dra27/cygsympathy) IIRC). Given that this is just for the duration of the runner, and it's not necessarily the user's default configuration, I think it's better not to change this?

Reply:
Ok, I've removed this commit.